deployment/HDI-Spark/Notebooks/README.md

```diff
@@ -0,0 +1,39 @@
+# Deploy SparkDotnet REPL to HDI Spark Cluster with Notebook Experience
+
+## Goal
+
+This documentation will elaborate the steps on how to deploy SparkDotnet REPL to HDI Spark Cluster and submit jobs through Jupyter Notebook using SparkDotnet.
+
+## Background
+
+We are currently using [dotnet-try](https://github.com/dotnet/try) as our dotnet REPL. ``` dotnet-try ``` can be used as a jupyter kernel and we implement it through [sparkmagic](https://github.com/jupyter-incubator/sparkmagic) and livy. To enable Jupyter Notebook Experience using SparkDotnet, this will include simple manual steps through [Ambari](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-manage-ambari) and [script actions](https://docs.microsoft.com/en-us/azure/hdinsight/hdinsight-hadoop-customize-cluster-linux) in HDI Spark Cluster.
+
+## Step
+
+### 1. Go to Ambari, manually stop Livy Server on head node.
+
+Details to be added.
+
+### 2. Run Script Actions on head node and worker node.
+
+Details to be added.
```

Comment:
Return to your HDI cluster in the Azure Portal, and select "Script actions" from the options on the left.

You'll submit 2 script actions: one to install Livy Spark Magic, and another to install sparkdotnet.

1) Install Livy Spark Magic
**Script Type:** custom
**Name:** install livy-spark-magic
**Bash script URI**: <uri where the first script is located, i.e.: https://mystorage.blob.core.windows.net/myname/install-livy-sparkmagic-hn.sh>
**Node type(s):** Head

2) Install sparkdotnet
**Script type:** custom
**Name:** install sparkdotnet
**Bash script URI:** <uri where the second script is located, i.e.: https://mystorage.blob.core.windows.net/myname/install-sparkdotnet-wn.sh>
**Node type(s):** Worker

**Note:** The second script action will likely take longer to run than the first.

After you see green check-marks next to the status of both script actions, you can move to the next step.